### PR TITLE
use updated dockerdns

### PR DESCRIPTION
**Pull request by davidsiaw**
Upstream dockerdns was using a very old version of the docker-py libraries that could not handle some of the weirder shit docker does. This fixes that.

To test this you need to update to the latest docker (26 and above)

just kaiser up any project.

### CODE REVIEW

**Comment by degikko** [comment]
@sikachu can you help us review this PR, please?

File: lib/kaiser/cli.rb [hunk 1]
```diff
@@ -582,8 +582,9 @@ def ensure_setup
           --name #{Config.config[:shared_names][:dns]}
           --network #{Config.config[:networkname]}
           --privileged
-          -v /var/run/docker.sock:/docker.sock:ro
-          phensley/docker-dns
+          -e DOCKER_HOST='unix:///var/run/docker.sock'
+          -v /var/run/docker.sock:/var/run/docker.sock:ro
+          davidsiaw/dockerdns
```
Comment (sikachu) [inline lib/kaiser/cli.rb:587]: Should we fork this into degica account instead?

**Review by sikachu — APPROVED** [review]
Just one question, but make sense to me otherwise.

File: lib/kaiser/cli.rb (hunk 1 above)
Comment (davidsiaw) [reply lib/kaiser/cli.rb:587]: we can. i can do that in another pr. i actually have a nice build system and everything on that now

**Pull request merged**